src/pages/iou/steps/MoneyRequstParticipantsPage/MoneyRequestParticipantsSelector.js

```diff
@@ -287,77 +297,107 @@ function MoneyRequestParticipantsSelector({
                     message="iou.error.splitBillMultipleParticipantsErrorMessage"
                 />
             )}
-            <Button
-                success
-                text={translate('iou.addToSplit')}
-                onPress={handleConfirmSelection}
-                pressOnEnter
-                isDisabled={shouldShowSplitBillErrorMessage}
-            />
+            <View style={[styles.pb5, styles.flexShrink0]}>
+                <PressableWithoutFeedback
+                    onPress={() => {
+                        Navigation.navigate(ROUTES.REFERRAL_DETAILS_MODAL.getRoute(referralContentType));
+                    }}
+                    style={[styles.p5, styles.w100, styles.br2, styles.highlightBG, styles.flexRow, styles.justifyContentBetween, styles.alignItemsCenter, {gap: 10}]}
+                    accessibilityLabel="referral"
+                    role={CONST.ACCESSIBILITY_ROLE.BUTTON}
+                >
+                    <Text>
+                        {translate(`referralProgram.${referralContentType}.buttonText1`)}
+                        <Text
+                            color={theme.success}
+                            style={styles.textStrong}
+                        >
+                            {translate(`referralProgram.${referralContentType}.buttonText2`)}
+                        </Text>
+                    </Text>
+                    <Icon
+                        src={Info}
+                        height={20}
+                        width={20}
+                    />
+                </PressableWithoutFeedback>
+            </View>
+
+            {shouldShowSplitButton && (
+                <Button
+                    success
+                    text={translate('iou.addToSplit')}
+                    onPress={handleConfirmSelection}
+                    pressOnEnter
+                    isDisabled={shouldShowSplitBillErrorMessage}
+                />
+            )}
         </View>
     );
 
+    const itemRightSideComponent = useCallback(
+        (item) => {
+            if (!isAllowedToSplit) {
+                return null;
+            }
+            return item.isSelected ? (
+                <PressableWithFeedback
+                    onPress={() => addParticipantToSelection(item)}
+                    disabled={item.isDisabled}
+                    role={CONST.ACCESSIBILITY_ROLE.CHECKBOX}
+                    accessibilityLabel={CONST.ACCESSIBILITY_ROLE.CHECKBOX}
+                    style={[styles.flexRow, styles.alignItemsCenter, styles.ml3]}
+                >
+                    <SelectCircle isChecked={item.isSelected} />
+                </PressableWithFeedback>
+            ) : (
+                <Button
+                    onPress={() => addParticipantToSelection(item)}
+                    style={[styles.pl2]}
+                    text={translate('iou.split')}
+                    small
+                />
+            );
```

Comment:
This way it should be more readable
```suggestion
            if (item.isSelected) {
                return (
                    <PressableWithFeedback
                        onPress={() => addParticipantToSelection(item)}
                        disabled={item.isDisabled}
                        role={CONST.ACCESSIBILITY_ROLE.CHECKBOX}
                        accessibilityLabel={CONST.ACCESSIBILITY_ROLE.CHECKBOX}
                        style={[styles.flexRow, styles.alignItemsCenter, styles.ml3]}
                    >
                        <SelectCircle isChecked={item.isSelected} />
                    </PressableWithFeedback>
                );
            }

            return (
                <Button
                    onPress={() => addParticipantToSelection(item)}
                    style={[styles.pl2]}
                    text={translate('iou.split')}
                    small
                />
            );
```

Reply:
I concur!